README.md

```diff
@@ -12,77 +12,71 @@ This package contains a set of tools to load and visualize the
-operational by default.
-
-This is because it requires dependencies that may be difficult to
-install on some systems. If you wish to have the interactive GUI
-working, install the
-[IPython Notebook](http://ipython.org/notebook.html)
-(now part of the [Jupyter project](http://jupyter.org/)) by hand:
-
-`conda install jupyter` (for the
-[Anaconda](https://www.continuum.io/downloads)
-distribution) or `pip install jupyter` (for the other Python distributions)
-
-NB: For [NERSC](http://www.nersc.gov/) users, it is not necessary to
-install the above package, as NERSC provides it when logging to
-[https://ipython.nersc.gov](https://ipython.nersc.gov).
-Therefore, NERSC users only need to install the `openPMD-viewer`
-package itself.
-
-## Usage
-
-The routines of openPMD-viewer can be used in two ways :
+The routines of `openPMD-viewer` can be used in two ways :
 
 - Use the **Python API**, in order to write a script that loads the
   data and produces a set of pre-defined plots.
 
-- Use the **interactive GUI inside the IPython Notebook**, in order to interactively
+- Use the **interactive GUI inside the Jupyter Notebook**, in order to interactively
 visualize the data.
 
+## Usage
+
 ### Tutorials
 
 The notebooks in the folder `tutorials/` demonstrate how to use both
 the API and the interactive GUI. You can view these notebooks online
-[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials),
-or, alternatively, you can run them on your local computer by typing:
-
-`ipython notebook tutorials/`
+[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials).
 
-NB: For [NERSC](http://www.nersc.gov/) users, you can run the tutorials on a
-remote machine by logging in at
-[https://ipython.nersc.gov](https://ipython.nersc.gov), and by
-navigating to your personal copy of the directory `openPMD-viewer/tutorials`.
+You can also download and run these notebooks on your local computer
+(when viewing the notebooks with the above link, click on `Raw` to be able to
+save them to your local computer).
 
 ### Notebook quick-starter
 
-If you wish to use the **interactive GUI**, the installation of `openPMD-viewer` provides
-a convenient executable which automatically
+If you wish to use the **interactive GUI**, the installation of
+`openPMD-viewer` provides a convenient executable which automatically
 **creates a new pre-filled notebook** and **opens it in a
 browser**. To use this executable, simply type in a regular terminal:
 
 `openPMD_notebook`
 
-(This executable is installed by default when running `python setup.py install`.)
+(This executable is installed by default.)
+
+## Installation
+
+### Installation on a local computer, using Anaconda
+
+On a local computer, it is recommended to install the [Anaconda
+distribution](https://www.continuum.io/downloads), which enables an easy
+installation of the dependencies of `openPMD-viewer`.
+
+Once Anaconda is installed, type
+```
+conda install scipy matplolib h5py jupyter

```

Comment:
I would generally advise the user to go for one install method, pip or conda based, when starting.

so just go for
```bash
conda install openPMD-viewer
```

below in that case and it will find and install the pip described dependencies but *through* conda.